### DISCUSSION TITLE
nextcloud-aio-nextcloud: cannot stop container: nextcloud-aio-nextcloud: tried to kill container, but did not receive an exit event

### DISCUSSION THREAD

**Discussion by pepper-lees** [nextcloud/all-in-one#2326] [Questions] [answered]
Hi,

I was having an issue where my proxy server started timing out, and my proxy server was receiving these errors:

```
Error: socket hang up
    at connResetException (internal/errors.js:628:14)
    at Socket.socketOnEnd (_http_client.js:499:23)
    at Socket.emit (events.js:412:35)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  code: 'ECONNRESET'
}
Error: connect ECONNREFUSED 127.0.0.1:11000
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1148:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 11000
}
```
(POTENTIALLY related to #2323)

so I attempted to restart my Nextcloud instance. Now, when attempting to stop the containers, the nextcloud-aio-nextcloud container is refusing to shut down. Docker is receiving this error:
```
nextcloud-aio-nextcloud: tried to kill container, but did not receive an exit event
```
Anything I can do to diagnose and remedy this? I was encountering the same issue when I was trying out ownCloud, and that issue was what caused me to switch over to Nextcloud. I've tried manually stopping the container via Docker, too, without success.

EDIT: additional info: Attempting to connect to the nextcloud-aio-nextcloud terminal results in this error:
```
$ docker exec -it nextcloud-aio-nextcloud bash
OCI runtime exec failed: exec failed: unable to start container process: error executing setns process: exit status 1: unknown
```
(I'm using Windows so there's no sudo before docker commands)

**Comment by pepper-lees** [comment, marked as answer]
In case this doesn't get a proper answer, but someone else encounters the same issue, here's what I did to get Nextcloud running again:

1. Rebooted the computer
2. Since Docker failed to restart after the computer rebooted, I checked [Docker's guide](https://docs.docker.com/desktop/troubleshoot/overview/) and used `docker.diagnose.exe check` to see what was wrong, and fixed as many of its suggestions as I could
3. Rebooted again
4. Docker managed to start, and nextcloud-aio-mastercontainer successfully started. Before I dared attempt to start containers again, I ran a backup
5. Started containers again, and thankfully it happened to run successfully.

**Comment by szaimen** [comment]
> nextcloud-aio-nextcloud: tried to kill container, but did not receive an exit event

This definitely looks like an issue of docker on windows itself ans is unrelated to AIO. So good to see that you did the recovering steps which apparently was the right solution.

**Reply by Tresillo2017** [reply]
It's happening to me on Ubuntu with the following error `Error response from daemon: cannot kill container: nextcloud-aio-nextcloud: tried to kill container, but did not receive an exit event`

**Reply by HammyHavoc** [reply]
Seeing this problem too: https://forums.unraid.net/topic/90003-cant-stop-docker-container-nextcloud-or-reboot-the-whole-system/page/2/?_fromLogin=1
